plugins/field-grid-dropdown/src/index.ts

```diff
@@ -196,38 +209,40 @@ export class FieldGridDropdown extends Blockly.FieldDropdown {
 Blockly.fieldRegistry.register('field_grid_dropdown', FieldGridDropdown);
 
 /**
- * CSS for slider field.
+ * CSS for grid field.
```

Comment:
The CSS changes are substantial and should be documented in the PR description (or possibly make this a breaking change).

Reply:
Done.